lib/_stream_writable.js

```diff
@@ -379,10 +379,18 @@ function clearBuffer(stream, state) {
     holder.entry = entry;
 
     var count = 0;
-    while (entry) {
-      buffer[count] = entry;
-      entry = entry.next;
-      count += 1;
+    if(state.objectMode){
```

Comment:
@cjihrig Thanks! Fix lint errors. But in `make test` i haven't seen any lint errors, only logs from  parallel/test-repl.